ComponentKit/DataSources/CKCollectionViewDataSource.mm

```diff
@@ -73,6 +73,11 @@ - (instancetype)init
   CK_NOT_DESIGNATED_INITIALIZER();
 }
 
+- (void)dealloc
+{
+    _collectionView.dataSource = nil;
```

Comment:
Oh, you were faster :) https://developer.apple.com/library/prerelease/ios/releasenotes/General/iOS90APIDiffs/Objective-C/UIKit.html